Lib/gftools/builder/__init__.py

```diff
@@ -249,11 +261,13 @@ def gen_stat_stylespace(self, stylespaceFile, varfonts):
                 stylespace, ttFont, additional_locations
             )
 
-    def build_static(self):
+    def build_static(self, cleanup=True):
```

Comment:
Do we need to run the cleanup step if we generate a VF as well?

Reply:
There's no need to create the `cleanup` function argument. Just use `self.config['cleanup']` in the function body instead.